### PR TITLE
GH 1423 - Policy PaaS should Disable Public Access for Container Apps

### PR DESCRIPTION
**Pull request by JamJarchitect**
## Overview/Summary

PR to fix Issue #1423 
Adds two new policy definitions to two initiatives to cover this off.

[AB#30888](https://dev.azure.com/CSUSolEng/2a9d5a5a-8998-4ad0-81c8-ef3045e4da97/_workitems/edit/30888)

## This PR fixes/adds/changes/removes

- Added ['Container Apps environment should disable public network access'](https://www.azadvertizer.net/azpolicyadvertizer/d074ddf8-01a5-4b5e-a2b8-964aed452c0a.html) to ['Deny-PublicPaaSEndpoints'.](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Deny-PublicPaaSEndpoints.html)
- Added ['Container Apps should only be accessible over HTTPS'](https://www.azadvertizer.net/azpolicyadvertizer/0e80e269-43a4-4ae9-b5bc-178126b8a5cb.html) to this ['Deny or Deploy and append TLS requirements and SSL enforcement on resources without Encryption in transit'.](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Enforce-EncryptTransit.html)

### Breaking Changes

1. N/A

## Testing Evidence

image

image

#### Azure Public

[Deploy To Azure](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FJamJarchitect%2FEnterprise-Scale%2Fhttps1423%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FJamJarchitect%2FEnterprise-Scale%2Fgh1423%2FeslzArm%2Feslz-portal.json)

#### Azure US Gov (Fairfax)
[Deploy To Azure](https://portal.azure.us/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FJamJarchitect%2FEnterprise-Scale%2Fgh1423%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FJamJarchitect%2FEnterprise-Scale%2Fgh1423%2FeslzArm%2Ffairfaxeslz-portal.json)

## As part of this Pull Request I have

- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [X] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [x] Performed testing and provided evidence.
- [X] Ensured [contribution guidance](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Contribution-Guide) is followed.
- [X] Updated relevant and associated documentation.
- [X] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -38,6 +38,13 @@ This article will be updated as and when changes are made to the above and anyth
 
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
+### October 2023
+
+#### Policy
+
+- Added [this policy definition](https://www.azadvertizer.net/azpolicyadvertizer/d074ddf8-01a5-4b5e-a2b8-964aed452c0a.html) to [this initiative.](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Deny-PublicPaaSEndpoints.html)
```
Comment (Springstone) [inline docs/wiki/Whats-new.md:45]: Please put in meaningful descriptions for policy definition and initiative. This is not readable currently, and requires clicking links to understand what we're addressing.

File: docs/wiki/Whats-new.md [hunk 2]
```diff
@@ -38,6 +38,13 @@ This article will be updated as and when changes are made to the above and anyth
...
+- Added [this policy definition](https://www.azadvertizer.net/azpolicyadvertizer/0e80e269-43a4-4ae9-b5bc-178126b8a5cb.html) to this [this initiative.](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Enforce-EncryptTransit.html)
```
Comment (Springstone) [inline docs/wiki/Whats-new.md:46]: Please put in meaningful descriptions for policy definition and initiative. This is not readable currently, and requires clicking links to understand what we're addressing.

**Review by Springstone — COMMENTED** [review]
Please also update the PR details with the same clarification on exactly what is changing.

**Comment by JamJarchitect** [comment]
@Springstone this is ready for review again please.

File: src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-EncryptTransit.json
Comment (Springstone) [inline src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-EncryptTransit.json:1]: Please increment initiative version.

**Review by Springstone — CHANGES_REQUESTED** [review]
If you could please address the ask, much appreciated.

**Pull request merged**